include/deal.II/base/table.h

```diff
@@ -538,16 +538,16 @@ class TableBase : public Subscriptor
    * in the equivalent of doing
    * @code
    *   Table<2,T> t;
-   *   for (unsigned int i=0; i<t.sizes()[0]; ++i)
-   *     for (unsigned int j=0; j<t.sizes()[1]; ++j)
+   *   for (size_type i=0; i<t.sizes()[0]; ++i)
+   *     for (size_type j=0; j<t.sizes()[1]; ++j)
```

Comment:
You are using `size_type` here in user code. I think you need to write this as
```
  typename Table<2,T>::size_type
```
or maybe just ignore the issue for the purposes of this example and stick with `unsigned int` :-)

Reply:
Yes, I switched back to `unsigned int`.